.github/copilot-instructions.md

```diff
@@ -0,0 +1,51 @@
+# Copilot Instructions for Sage Monorepo
+
+## Project Overview
+
+This is a TypeScript-based monorepo for Sage Bionetworks projects. It contains multiple applications and libraries related to scientific research, data visualization, and biomedical informatics. The codebase uses Angular, React, and other modern web technologies.
```

Comment:
In order to not decrease the quality of the responses to non-TypeScript projects, please consider including the following information:

- This monorepo is multi-language and includes projects in TypeScript, Java, Python and R.
- Consider mentioning that this is an Nx monorepo.